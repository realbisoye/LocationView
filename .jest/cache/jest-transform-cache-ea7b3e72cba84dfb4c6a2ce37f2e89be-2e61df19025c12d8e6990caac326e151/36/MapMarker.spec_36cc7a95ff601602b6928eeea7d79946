838108dfde080fdebf0122149fc4e72f
"use strict";var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");var _react = _interopRequireDefault(require("react"));
var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));
var _mapMarker = _interopRequireDefault(require("../../app/components/mapMarker"));var _jsxFileName = "/Users/bisoyeatolagbe/projects/bisoye/LocationView/__tests__/components/MapMarker.spec.js";

describe('MapMarker', function () {
  it('renders map marker', function () {
    var rendered = _reactTestRenderer.default.create(
    _react.default.createElement(_mapMarker.default, {
      id: '1',
      latitude: 40.785091,
      longitude: -73.968285,
      description: 'Central Park playground',
      title: 'Central Park', __source: { fileName: _jsxFileName, lineNumber: 8 } }));



    expect(rendered).toMatchSnapshot();
  });
});