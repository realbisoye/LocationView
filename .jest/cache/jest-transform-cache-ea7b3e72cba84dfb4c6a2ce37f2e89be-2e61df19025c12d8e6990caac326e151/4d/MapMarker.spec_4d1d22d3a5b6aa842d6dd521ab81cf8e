455e205bd8346b24ce69741cea2be46c
"use strict";var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");var _react = _interopRequireDefault(require("react"));
var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));
var _mapMarker = _interopRequireDefault(require("../../app/components/mapMarker"));var _jsxFileName = "/Users/bisoyeatolagbe/projects/bisoye/LocationView/__tests__/components/MapMarker.spec.js";

describe('Marker', function () {
  it('renders Marker', function () {
    var rendered = _reactTestRenderer.default.create(
    _react.default.createElement(_mapMarker.default, {
      id: 1,
      latitude: 40.785091,
      longitude: -73.968285,
      description: 'Central Park playground',
      title: 'Central Park', __source: { fileName: _jsxFileName, lineNumber: 8 } }));



    expect(rendered).toMatchSnapshot();
  });
});