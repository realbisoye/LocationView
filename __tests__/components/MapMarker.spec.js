import React from 'react'
import renderer from 'react-test-renderer'
import MapMarker from '../../app/components/mapMarker'

describe('MapMarker', () => {
  it('renders map marker', () => {
    const rendered = renderer.create(
      <MapMarker
        id={'1'}
        latitude={40.785091}
        longitude={-73.968285}
        description={'Central Park playground'}
        title={'Central Park'}
      />,
    )

    expect(rendered).toMatchSnapshot()
  })
})
