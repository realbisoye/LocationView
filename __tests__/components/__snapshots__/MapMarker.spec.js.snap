// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapMarker renders map marker 1`] = `"Marker"`;
